templates/shared/forms/form-fields.html

```diff
@@ -0,0 +1,208 @@
+                  {% if field.type == "country" %}
+                    {% include "shared/forms/_country.html" %}
+                  {% else %}
+                    <li class="p-list__item">
+                      {% if field.type == "text" or field.type == "tel" or field.type == "email" %}
+                        <label {% if field.isRequired %}class="is-required"{% endif %}
+                               for="{{ field.id }}">
+                          {{ field.label
+                          }}:
+                        </label>
+                        <input {% if field.isRequired %}required{% endif %}
+                               id="{{ field.id }}"
+                               name="{{ field.id }}"
+                               maxlength="255"
+                               type="{{ field.type }}"
+                               {% if field.type=="email" %}pattern="^[^ ]+@[^ ]+\.[a-z]{2,26}$"{% endif %} />
+                      {% elif field.type == "long-text" %}
+                        <textarea {% if fieldset.isRequired %}required{% endif %}
+                                  id="{{field.id}}"
+                                  rows="5"
+                                  maxlength="2000"></textarea>
+                      {% elif field.type == "radio" %}
+                        <label class="p-radio">
+                          <input {% if fieldset.isRequired %}required{% endif %}
+                                 class="p-radio__input"
+                                 type="radio"
+                                 aria-labelledby="{{ field.id }}"
+                                 name="{{ fieldset.inputName }}"
+                                 value="{{ field.value }}" />
+                          <span class="p-radio__label" id="{{ field.id }}">{{ field.label }}</span>
+                        </label>
+                      {% elif field.type == "checkbox" %}
+                        <label class="p-checkbox">
+                          <input class="p-checkbox__input js-checkbox-visibility"
+                                 type="checkbox"
+                                 aria-labelledby="{{ field.label }}"
+                                 value="{{ field.value }}" />
+                          <span class="p-checkbox__label" id="{{ field.id }}">{{ field.label }}</span>
+                        </label>
+                      {% elif field.fieldTitle is defined and field.fieldTitle|length > 0 %}
+                        <div class="p-section--shallow">
+                          <strong>{{ field.fieldTitle }}</strong>
+                          {% for option in field.options %}
+                            <label class="p-checkbox">
+                              <input {% if field.fieldTitle=="Other" %} class="p-checkbox__input js-checkbox-visibility__other" {%
+                                else %} class="p-checkbox__input js-checkbox-visibility" {% endif %} type="checkbox"
```

Comment:
```suggestion
                              <input class="p-checkbox__input js-checkbox-visibility{% if field.fieldTitle=="Other" %}__other"{% endif %}" type="checkbox"
```
This conditional statement can be written in a cleaner way